<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayWindow</class>
 <widget class="QMainWindow" name="PlayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>ChessGame</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;asgafgdfg&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background:rgb(50,46,43);
}
QPushButton{
	color:rgb(162,150,135);
	background-color:rgb(255,255,255);
	font-size:60px;
	border-radius:10px;
}
QPushButton::hover{
	background-color:rgb(222,222,222);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="chess_let">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>44</y>
      <width>653</width>
      <height>105</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>18thCentury</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
	color: rgb(162,150,135);
</string>
    </property>
    <property name="text">
     <string>Шахматы</string>
    </property>
   </widget>
   <widget class="QLabel" name="chess_img">
    <property name="geometry">
     <rect>
      <x>-156</x>
      <y>18</y>
      <width>670</width>
      <height>740</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>chess.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="backToMenuButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>550</y>
      <width>221</width>
      <height>82</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startTaskButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>450</y>
      <width>221</width>
      <height>82</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Начать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="taskButton_1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>300</width>
      <height>82</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Задача 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="taskButton_6">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>170</y>
      <width>300</width>
      <height>82</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Задача 6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="taskButton_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>300</width>
      <height>82</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Задача 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="taskButton_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>300</width>
      <height>82</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Задача 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="taskButton_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>470</y>
      <width>300</width>
      <height>82</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Задача 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="taskButton_7">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>270</y>
      <width>300</width>
      <height>82</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Задача 7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="taskButton_5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>570</y>
      <width>300</width>
      <height>82</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Задача 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="taskButton_8">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>370</y>
      <width>300</width>
      <height>82</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Задача 8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="taskButton_9">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>470</y>
      <width>300</width>
      <height>82</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Задача 9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="taskButton_10">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>570</y>
      <width>300</width>
      <height>82</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Задача 10</string>
    </property>
   </widget>
   <zorder>chess_img</zorder>
   <zorder>chess_let</zorder>
   <zorder>backToMenuButton</zorder>
   <zorder>startTaskButton</zorder>
   <zorder>taskButton_1</zorder>
   <zorder>taskButton_6</zorder>
   <zorder>taskButton_3</zorder>
   <zorder>taskButton_2</zorder>
   <zorder>taskButton_4</zorder>
   <zorder>taskButton_7</zorder>
   <zorder>taskButton_5</zorder>
   <zorder>taskButton_8</zorder>
   <zorder>taskButton_9</zorder>
   <zorder>taskButton_10</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
